### PR TITLE
W1 - D5 Activity Fragment

### PR DESCRIPTION
**Pull request by at-hungtran2**

### CODE REVIEW

**Comment by at-hungtran2** [comment]
@at-hanhnguyen2  please review it,tks

File: app/src/at-hungtran/AndroidManifest.xml [hunk 1]
```diff
@@ -9,13 +9,14 @@
...
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity android:name=".LogActivity">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".ViewActivity"></activity>
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/AndroidManifest.xml]: @at-hungtran2 : change ViewActivity -> InformationActivity

File: app/src/at-hungtran/java/asiantech/internship/summer/LogActivity.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+public class LogActivity extends AppCompatActivity {
+    Button btn;
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/LogActivity.java]: @at-hungtran2 : set name view follow convention and clear mean

File: app/src/at-hungtran/java/asiantech/internship/summer/LogActivity.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        setContentView(R.layout.activity_log);
+        FragmentTransaction fragmentTransaction = getSupportFragmentManager().beginTransaction();
+        fragmentTransaction.replace(R.id.fragment, new LoginFragment());
+        fragmentTransaction.commit();
+        btn = findViewById(R.id.btnBack);
+        btn.setVisibility(View.GONE);
+        backFragment();
+    }
+
+    void replaceFragmemnt() {
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/LogActivity.java]: @at-hungtran2 : replaceFragmemnt() -> replaceFragment()

File: app/src/at-hungtran/java/asiantech/internship/summer/LogActivity.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        FragmentManager fragmentManager = getSupportFragmentManager();
+        FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();
+        fragmentTransaction.setCustomAnimations(R.anim.anim_right_to_left,
+                R.anim.slide_out_left, R.anim.anim_left_to_right,
+                R.anim.slide_out_right);
+        setTextForToolBar("SIGN UP");
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/LogActivity.java]: @at-hungtran2 : move text to Strings.xml file

File: app/src/at-hungtran/java/asiantech/internship/summer/LogActivity.java [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        fragmentTransaction.addToBackStack(null);
+        fragmentTransaction.commit();
+
+    }
+
+    public void onButton() {
+        btn.setVisibility(View.VISIBLE);
+    }
+
+    public void setTextForToolBar(String mText) {
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/LogActivity.java]: @at-hungtran2 : change mText -> title

File: app/src/at-hungtran/java/asiantech/internship/summer/LogActivity.java [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+        tv = findViewById(R.id.tvToolBar);
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/LogActivity.java]: @at-hungtran2 : move findViewById to onCreate()

File: app/src/at-hungtran/java/asiantech/internship/summer/LogActivity.java [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+        btn.setOnClickListener(v -> {
+            onBackPressed();
+        });
+    }
+
+    @Override
+    public void onBackPressed() {
+        super.onBackPressed();
+        Fragment fragment = getSupportFragmentManager().findFragmentById(R.id.fragment);
+        if (fragment instanceof LoginFragment) {
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/LogActivity.java]: @at-hungtran2 : move string to Strings.xml file

File: app/src/at-hungtran/java/asiantech/internship/summer/LoginFragment.java [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+import java.util.regex.Pattern;
+
+public class LoginFragment extends Fragment {
+    private EditText mEdtEmail;
+    private EditText mEdtPwd;
+
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        Log.d("xx", "onCreateView: ");
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/LoginFragment.java]: @at-hungtran2 : delete log

File: app/src/at-hungtran/java/asiantech/internship/summer/LoginFragment.java [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+        initView(view);
+        return view;
+    }
+
+    private void initView(View view) {
+        Button btnSignUp = view.findViewById(R.id.btnSignUp);
+        Button mBtnLogin = view.findViewById(R.id.btnLogin);
+        mEdtPwd = view.findViewById(R.id.edtPwd);
+        mEdtEmail = view.findViewById(R.id.edtEmail);
+        btnSignUp.setOnClickListener(new View.OnClickListener() {
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/LoginFragment.java]: @at-hungtran2 : use lambada to fix warning

File: app/src/at-hungtran/java/asiantech/internship/summer/LoginFragment.java [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+            public void onClick(View view) {
+                ((LogActivity) getActivity()).replaceFragmemnt();
+            }
+        });
+        mBtnLogin.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                if (checkEmailPassword()) {
+                    Intent i = new Intent(getActivity(), ViewActivity.class);
+                    i.putExtra("email", mEdtEmail.getText().toString());
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/LoginFragment.java]: @at-hungtran2 : define key email and password
ex: private static final String KEY_EMAIL = "email"
private static final String KEY_PASS = "password"

File: app/src/at-hungtran/java/asiantech/internship/summer/LoginFragment.java [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+                    i.putExtra("password", mEdtPwd.getText().toString());
+                    startActivity(i);
+                } else {
+                    Toast.makeText(getActivity(), "Please check your email and password",
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/LoginFragment.java]: @at-hungtran2 : move string to Strings.xml file

File: app/src/at-hungtran/java/asiantech/internship/summer/LoginFragment.java [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+                            Toast.LENGTH_SHORT).show();
+                }
+            }
+        });
+    }
+
+    private boolean checkEmailPassword() {
+        Button btnLogin = getView().findViewById(R.id.btnLogin);
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/LoginFragment.java]: @at-hungtran2 : remove this line if you don't use it

File: app/src/at-hungtran/java/asiantech/internship/summer/LoginFragment.java [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+        String mPassword = mEdtPwd.getText().toString();
+        String mEmail = mEdtEmail.getText().toString();
+        return isValidPassword(mPassword) && isValidEmail(mEmail);
+    }
+
+    private boolean isValidEmail(String mEmail) {
+        Pattern pattern;
+        Matcher matcher;
+        final String EMAIL_PATTERN = "^[_A-Za-z0-9-]+(\\.[_A-Za-z0-9-]+)*@[A-Za-z0-9]+(\\.[A-Za-z0-9]+)*(\\.[A-Za-z]{2,})$";
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/LoginFragment.java]: @at-hungtran2 : please define on top class

File: app/src/at-hungtran/java/asiantech/internship/summer/LoginFragment.java [hunk 14]
```diff
@@ -0,0 +1,82 @@
...
+        pattern = Pattern.compile(EMAIL_PATTERN);
+        matcher = pattern.matcher(mEmail);
+        return matcher.matches();
+    }
+
+    private boolean isValidPassword(String mPassword) {
+        Pattern pattern;
+        Matcher matcher;
+        final String PASSWORD_PATTERN = "^(?=.*[0-9])(?=.*[a-zA-Z]).{6,18}$";
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/LoginFragment.java]: @at-hungtran2 : please define on top class

File: app/src/at-hungtran/java/asiantech/internship/summer/SignUpFragment.java [hunk 15]
```diff
@@ -0,0 +1,77 @@
...
+import android.widget.EditText;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class SignUpFragment extends Fragment {
+    private EditText mEdtSUEmail;
+    private EditText mEdtSUPwd;
+    private EditText mEdtSURePwd;
+    private View view;
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/SignUpFragment.java]: @at-hungtran2 : remove this line

File: app/src/at-hungtran/java/asiantech/internship/summer/SignUpFragment.java [hunk 16]
```diff
@@ -0,0 +1,77 @@
...
+        view = inflater.inflate(R.layout.fragment_sign_up, container, false);
+        initView(view);
+        return view;
+    }
+
+    private void initView(View view) {
+        mEdtSUEmail = view.findViewById(R.id.edtSUEmail);
+        mEdtSUPwd = view.findViewById(R.id.edtSUPwd);
+        mEdtSURePwd = view.findViewById(R.id.edtSURePwd);
+        Button mBtnSUSignUp = view.findViewById(R.id.btnSUSignUp);
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/SignUpFragment.java]: @at-hungtran2 : mBtnSUSignUp -> btnSignUp

File: app/src/at-hungtran/java/asiantech/internship/summer/SignUpFragment.java [hunk 17]
```diff
@@ -0,0 +1,77 @@
...
+        mBtnSUSignUp.setOnClickListener(new View.OnClickListener() {
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/SignUpFragment.java]: @at-hungtran2 : use lambda to fix warning

File: app/src/at-hungtran/java/asiantech/internship/summer/SignUpFragment.java [hunk 18]
```diff
@@ -0,0 +1,77 @@
...
+                    fragmentTransaction.replace(R.id.fragment, new LoginFragment());
+                    fragmentTransaction.addToBackStack(null);
+                    fragmentTransaction.commit();
+                }
+            }
+        });
+    }
+
+    private boolean checkEmailPassword() {
+        Button btnLogin = view.findViewById(R.id.btnLogin);
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/SignUpFragment.java]: @at-hungtran2 : remove this line if you don't use it

File: app/src/at-hungtran/java/asiantech/internship/summer/SignUpFragment.java [hunk 19]
```diff
@@ -0,0 +1,77 @@
...
+        String mPassword = mEdtSUPwd.getText().toString();
+        String mRePassword = mEdtSURePwd.getText().toString();
+        String mEmail = mEdtSUEmail.getText().toString();
+        return isValidPassword(mPassword) && isValidEmail(mEmail) && mRePassword.equals(mPassword);
+    }
+
+    private boolean isValidEmail(String mEmail) {
+        Pattern pattern;
+        Matcher matcher;
+        final String EMAIL_PATTERN = "^[_A-Za-z0-9-]+(\\.[_A-Za-z0-9-]+)*@[A-Za-z0-9]+(\\.[A-Za-z0-9]+)*(\\.[A-Za-z]{2,})$";
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/SignUpFragment.java]: @at-hungtran2 : set on top class

File: app/src/at-hungtran/java/asiantech/internship/summer/SignUpFragment.java [hunk 20]
```diff
@@ -0,0 +1,77 @@
...
+        pattern = Pattern.compile(EMAIL_PATTERN);
+        matcher = pattern.matcher(mEmail);
+        return matcher.matches();
+    }
+
+    private boolean isValidPassword(String mPassword) {
+        Pattern pattern;
+        Matcher matcher;
+        final String PASSWORD_PATTERN = "^(?=.*[0-9])(?=.*[a-zA-Z]).{6,18}$";
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/SignUpFragment.java]: @at-hungtran2 : set on top class

File: app/src/at-hungtran/java/asiantech/internship/summer/ViewActivity.java [hunk 21]
```diff
@@ -0,0 +1,21 @@
...
+
+public class ViewActivity extends AppCompatActivity {
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_view);
+        TextView mTvEmail = findViewById(R.id.tvEmail);
+        TextView mTvPassword = findViewById(R.id.tvPassword);
+        Intent intent = getIntent();
+        String mValueEmail = intent.getStringExtra("email");
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/ViewActivity.java]: @at-hungtran2 : define key email and password

File: app/src/at-hungtran/res/drawable/border.xml [hunk 22]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <!-- Set màu nền-->
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/res/drawable/border.xml]: @at-hungtran2 : don't comment by vietnamese

File: app/src/at-hungtran/res/layout/activity_log.xml [hunk 23]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/res/layout/activity_log.xml]: @at-hungtran2 : activity_log -> activity_login

File: app/src/at-hungtran/res/layout/activity_log.xml [hunk 24]
```diff
@@ -0,0 +1,31 @@
...
+        <TextView
+            android:id="@+id/tvToolBar"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="30dp"
+            android:text="@string/login" />
+    </android.support.v7.widget.Toolbar>
+
+    <android.support.v7.widget.ContentFrameLayout
+        android:id="@+id/fragment"
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/res/layout/activity_log.xml]: @at-hungtran2 : fragment -> frContainer

File: app/src/at-hungtran/res/layout/activity_view.xml [hunk 25]
```diff
@@ -0,0 +1,37 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/res/layout/activity_view.xml]: @at-hungtran2 : activity_view -> activity_information

File: app/src/at-hungtran/res/layout/activity_view.xml [hunk 26]
```diff
@@ -0,0 +1,37 @@
...
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/colorPrimary"
+    android:orientation="vertical"
+    tools:context="asiantech.internship.summer.SignUpFragment">
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/res/layout/activity_view.xml]: @at-hungtran2 : remove this line

File: app/src/at-hungtran/res/layout/activity_view.xml [hunk 27]
```diff
@@ -0,0 +1,37 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:layout_marginLeft="30dp"
+        android:layout_marginTop="20dp"
+        android:layout_marginRight="30dp"
+        android:background="@drawable/border"
+        android:gravity="center"
+        android:paddingLeft="20dp"
+        android:textColor="#FFF"
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/res/layout/activity_view.xml]: @at-hungtran2 : color set in colors.xml file

File: app/src/at-hungtran/res/layout/activity_view.xml [hunk 28]
```diff
@@ -0,0 +1,37 @@
...
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/res/layout/activity_view.xml]: @at-hungtran2 : color set in colors.xml file

File: app/src/at-hungtran/res/layout/fragment_login.xml [hunk 29]
```diff
@@ -0,0 +1,59 @@
...
+    <EditText
+        android:id="@+id/edtEmail"
+        android:layout_width="200dp"
+        android:layout_height="50dp"
+        android:layout_gravity="center"
+        android:layout_marginTop="100dp"
+        android:background="@drawable/border"
+        android:hint="@string/email"
+        android:paddingLeft="20dp"
+        android:textColor="#FFF"
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/res/layout/fragment_login.xml]: @at-hungtran2 : color set in colors.xml file

File: app/src/at-hungtran/res/layout/fragment_login.xml [hunk 30]
```diff
@@ -0,0 +1,59 @@
...
+        android:id="@+id/edtPwd"
+        android:layout_width="200dp"
+        android:layout_height="50dp"
+        android:layout_gravity="center"
+        android:layout_marginTop="20dp"
+        android:background="@drawable/border"
+        android:hint="@string/password"
+        android:inputType="textPassword"
+        android:paddingLeft="20dp"
+        android:textColor="#FFF"
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/res/layout/fragment_login.xml]: @at-hungtran2 : color set in colors.xml file

File: app/src/at-hungtran/res/layout/fragment_login.xml [hunk 31]
```diff
@@ -0,0 +1,59 @@
...
+    <Button
+        android:id="@+id/btnSignUp"
+        android:layout_width="200dp"
+        android:layout_height="50dp"
+        android:layout_gravity="center"
+        android:layout_marginTop="30dp"
+        android:background="@color/colorPrimary"
+        android:text="@string/singup"
+        android:textAllCaps="false"
+        android:textColor="#FFF"
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/res/layout/fragment_login.xml]: @at-hungtran2 : color set in colors.xml file

File: app/src/at-hungtran/res/layout/fragment_login.xml [hunk 32]
```diff
@@ -0,0 +1,59 @@
...
+
+    <Button
+        android:id="@+id/btnLogin"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:layout_marginTop="50dp"
+        android:background="@color/colorPrimary"
+        android:text="@string/login"
+        android:textColor="#FFF"
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/res/layout/fragment_login.xml]: @at-hungtran2 : color set in colors.xml file

File: app/src/at-hungtran/res/layout/fragment_sign_up.xml [hunk 33]
```diff
@@ -0,0 +1,59 @@
...
+        android:id="@+id/edtSUEmail"
+        android:layout_width="200dp"
+        android:layout_height="50dp"
+        android:layout_gravity="center"
+        android:layout_marginTop="100dp"
+        android:background="@drawable/border"
+        android:hint="@string/email"
+        android:importantForAutofill="no"
+        android:paddingLeft="20dp"
+        android:textColor="#FFF"
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/res/layout/fragment_sign_up.xml]: @at-hungtran2 : color set in colors.xml file

File: app/src/at-hungtran/res/layout/fragment_sign_up.xml [hunk 34]
```diff
@@ -0,0 +1,59 @@
...
+
+    <EditText
+        android:id="@+id/edtSUEmail"
+        android:layout_width="200dp"
+        android:layout_height="50dp"
+        android:layout_gravity="center"
+        android:layout_marginTop="100dp"
+        android:background="@drawable/border"
+        android:hint="@string/email"
+        android:importantForAutofill="no"
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/res/layout/fragment_sign_up.xml]: @at-hungtran2 : android:importantForAutofill="no"    ?

File: app/src/at-hungtran/res/layout/fragment_sign_up.xml [hunk 35]
```diff
@@ -0,0 +1,59 @@
...
+        android:id="@+id/edtSUPwd"
+        android:layout_width="200dp"
+        android:layout_height="50dp"
+        android:layout_gravity="center"
+        android:layout_marginTop="20dp"
+        android:background="@drawable/border"
+        android:hint="@string/password"
+        android:inputType="textPassword"
+        android:paddingLeft="20dp"
+        android:textColor="#FFF"
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/res/layout/fragment_sign_up.xml]: @at-hungtran2 : color set in colors.xml file

File: app/src/at-hungtran/res/layout/fragment_sign_up.xml [hunk 36]
```diff
@@ -0,0 +1,59 @@
...
+        android:id="@+id/edtSURePwd"
+        android:layout_width="200dp"
+        android:layout_height="50dp"
+        android:layout_gravity="center"
+        android:layout_marginTop="20dp"
+        android:background="@drawable/border"
+        android:hint="@string/confirm_password"
+        android:inputType="textPassword"
+        android:paddingLeft="20dp"
+        android:textColor="#FFF"
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/res/layout/fragment_sign_up.xml]: @at-hungtran2 : color set in colors.xml file

File: app/src/at-hungtran/res/layout/fragment_sign_up.xml [hunk 37]
```diff
@@ -0,0 +1,59 @@
...
+
+    <Button
+        android:id="@+id/btnSUSignUp"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:layout_marginTop="50dp"
+        android:background="@color/colorPrimary"
+        android:text="@string/sign_up"
+        android:textColor="#FFF"
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/res/layout/fragment_sign_up.xml]: @at-hungtran2 : color set in colors.xml file

File: app/src/at-hungtran/res/layout/fragment_sign_up.xml [hunk 38]
```diff
@@ -0,0 +1,59 @@
...
+        android:layout_marginTop="20dp"
+        android:background="@drawable/border"
+        android:hint="@string/confirm_password"
+        android:inputType="textPassword"
+        android:paddingLeft="20dp"
+        android:textColor="#FFF"
+        tools:ignore="Autofill,TextFields" />
+
+    <Button
+        android:id="@+id/btnSUSignUp"
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/res/layout/fragment_sign_up.xml]: @at-hungtran2 : btnSUSignUp -> btnSignUp

File: app/src/at-hungtran/java/asiantech/internship/summer/LoginFragment.java [hunk 39]
```diff
@@ -0,0 +1,81 @@
...
+                startActivity(i);
+            } else {
+                Toast.makeText(getActivity(), getString(R.string.pleasecheckemailpassword),
+                        Toast.LENGTH_SHORT).show();
+            }
+        });
+    }
+
+    private boolean checkEmailPassword() {
+        String mPassword = mEdtPwd.getText().toString();
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/LoginFragment.java]: @at-hungtran2 : mPassword -> password, mEmail -> email

File: app/src/at-hungtran/java/asiantech/internship/summer/SignUpFragment.java [hunk 40]
```diff
@@ -0,0 +1,77 @@
...
+                fragmentTransaction.setCustomAnimations(R.anim.anim_right_to_left, R.anim.anim_left_to_right);
+                fragmentTransaction.replace(R.id.frContainer, new LoginFragment());
+                fragmentTransaction.addToBackStack(null);
+                fragmentTransaction.commit();
+            }
+        });
+    }
+
+    private boolean checkEmailPassword() {
+        String mPassword = mEdtPwd.getText().toString();
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/java/asiantech/internship/summer/SignUpFragment.java]: @at-hungtran2 : mPassword -> password, mEmail -> email

File: app/src/at-hungtran/res/layout/activity_infomation.xml [hunk 41]
```diff
@@ -0,0 +1,37 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:layout_marginLeft="30dp"
+        android:layout_marginTop="20dp"
+        android:layout_marginRight="30dp"
+        android:background="@drawable/border"
+        android:gravity="center"
+        android:paddingLeft="20dp"
+        android:textColor="#FFF"
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/res/layout/activity_infomation.xml]: @at-hungtran2 : move color to colors.xml file

File: app/src/at-hungtran/res/layout/activity_infomation.xml [hunk 42]
```diff
@@ -0,0 +1,37 @@
...
```
Comment (at-hanhnguyen2) [inline app/src/at-hungtran/res/layout/activity_infomation.xml]: @at-hungtran2 : move color to colors.xml file

File: app/src/at-hungtran/java/asiantech/internship/summer/activityfragment/LoginFragment.java [hunk 43]
```diff
@@ -0,0 +1,85 @@
...
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import asiantech.internship.summer.R;
+import asiantech.internship.summer.activityfragment.InfomationActivity;
```
Comment (at-dev-support-01) [inline app/src/at-hungtran/java/asiantech/internship/summer/activityfragment/LoginFragment.java]: WARNING: [ImportFromSamePackage] No need to import a type that lives in the same package
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/imports.html#ImportFromSamePackage

File: app/src/at-hungtran/java/asiantech/internship/summer/activityfragment/LoginFragment.java [hunk 44]
```diff
@@ -0,0 +1,85 @@
...
+import asiantech.internship.summer.activityfragment.LoginActivity;
```
Comment (at-dev-support-01) [inline app/src/at-hungtran/java/asiantech/internship/summer/activityfragment/LoginFragment.java]: WARNING: [ImportFromSamePackage] No need to import a type that lives in the same package
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/imports.html#ImportFromSamePackage

File: app/src/at-hungtran/java/asiantech/internship/summer/activityfragment/SignUpFragment.java [hunk 45]
```diff
@@ -0,0 +1,80 @@
...
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import asiantech.internship.summer.R;
+import asiantech.internship.summer.activityfragment.LoginFragment;
```
Comment (at-dev-support-01) [inline app/src/at-hungtran/java/asiantech/internship/summer/activityfragment/SignUpFragment.java]: WARNING: [ImportFromSamePackage] No need to import a type that lives in the same package
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/imports.html#ImportFromSamePackage

**Pull request merged**